experimental/packages/opentelemetry-sdk-metrics-base/test/export/ConsoleMetricExporter.test.ts

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import * as metrics from '@opentelemetry/api-metrics';
+import { ExportResult } from '@opentelemetry/core';
+import { ConsoleMetricExporter } from '../../src/export/ConsoleMetricExporter';
+import { PeriodicExportingMetricReader } from '../../src/export/PeriodicExportingMetricReader';
+import { ResourceMetrics } from '../../src/export/MetricData';
+import { MeterProvider } from '../../src/MeterProvider';
+import { defaultResource } from '../util';
+import * as assert from 'assert';
+import * as sinon from 'sinon';
+
+
+async function waitForNumberOfExports(exporter: sinon.SinonSpy<[metrics: ResourceMetrics, resultCallback: (result: ExportResult) => void], void>, numberOfExports: number): Promise<ResourceMetrics[]> {
+  if (numberOfExports <= 0) {
+    throw new Error('numberOfExports must be greater than or equal to 0');
+  }
+
+  let totalExports = 0;
+  while (totalExports < numberOfExports) {
+    await new Promise(resolve => setTimeout(resolve, 20));
+    totalExports = exporter.callCount;
+  }
+
+  return [];
```

Comment:
The return value seems to be always `[]`. Could we make this function return `Promise<void>` instead? :slightly_smiling_face: